doc/v1.3/partials/jobfile/run-log.html

```diff
@@ -2,20 +2,16 @@
   <h3>Keeping a Log of Job Runs</h3>
 
   <p>
-    The command
-    <code>jobber log</code>
-    prints out info about previous runs. This info &mdash; the
-    <dfn>run log</dfn>
-    &mdash; must be kept somewhere, and you can control this in the
-    <code>prefs</code>
+    <code>jobber log</code> command prints info about 100 last runs.
+    This info &mdash; <dfn>run log</dfn> &mdash; is kept in memory by
+    default, but you can change this in the <code>prefs</code>
```

Comment:
How about this:

```
<p>The <code>jobber log</code> command prints the <dfn>run log</dfn> &mdash; i.e., a log 
containing info about previous runs.  By default, the run log only contains the 100 most recent 
runs, and it is not written to disk, which means that it will be lost when the Jobber service stops 
or restarts.</p>
```

Reply:
LGTM. If `jobber` doesn't expose any other logs. then the `run` word can be omitted.

Reply:
Cool.  Would you mind making those changes in this PR?